phpstan.dist.neon

```diff
@@ -22,4 +22,4 @@ parameters:
             message: "#Method .* return type has no value type specified in iterable type array#"
             path: webapp/src/DataFixtures/Test
 includes:
-    - lib/vendor/phpstan/phpstan-doctrine/extension.neon
+    - webapp/vendor/phpstan/phpstan-doctrine/extension.neon
```

Comment:
We might (later) want to move this file and the phpunit file to webapp as well? Since they all test webapp only anyway.

Reply:
It would be nice to also run PHPStan on the judgedaemon but I suspect we'll never reach that step so fine by me.